src/system.h

```diff
@@ -17,12 +15,18 @@ struct Image {
 
     long last_display = 0;
 
-    Image(wxBitmap _bm, uint64_t _hash, double _sc, vector<uint8_t> &&pd, char iti)
-        : bm_orig(_bm), hash(_hash), display_scale(_sc), image_data(std::move(pd)), image_type(iti) {}
+    Image(const wxBitmap &bm, uint64_t _hash, double _sc, vector<uint8_t> &&pd, char iti)

```

Comment:
Note that the caller for this constructor is often `AddImageToList`, which got an `Image` and `vector` from loading, which then converts it to a bitmap, an you then convert that bitmap back.. this constructor should not take a bitmap.

Also you're copying `image_data` from the vector, then overwriting it again with the bitmap???? Please be sure you understand the code you modify precisely.

Reply:
I see your point. This PR is only the half way. The other half way comes (hopefully) with https://github.com/aardappel/treesheets/pull/367/commits/aca1a28dd69aa3a4e99de341fb8e9a2f5fd0260c

Reply:
PRs should be self contained and do only one thing. They should not contain unrelated changes or rely on changes in future PRs. Can you make this more reviewable? I can't know what you're all still planning to do.

Reply:
I think that the PR should be now ready. It should do one thing: remove bm_orig with all its consequences
Is this ok for you?